appinfo/info.xml

```diff
@@ -39,7 +39,8 @@
     </types>
 
 	<dependencies>
-		<nextcloud min-version="20" max-version="24" />
+		<php min-version="7.3" max-version="8.0" />
```

Comment:
is this something we should do globally? Like contacts doesn't really have a specific need to define it's own php version rather than those server provides?

I have no expertise on the matter, genuinely wondering if we should apply this globally to our apps :)

Reply:
We recently screwed up due to the lack of this. It's not a must-have. It just makes the php support range more explicit.